src/python/grpcio/grpc/_cython/_cygrpc/event.pyx.pxi

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+cdef class BaseEvent:
```

Comment:
This is redundant since it's empty. You only need the stub in the `pxd.pxi` file.